base-notebook/Dockerfile.ppc64le.patch

```diff
@@ -81,14 +81,14 @@
  
 -# Install Jupyter Notebook and Hub
 -RUN conda install --quiet --yes \
--    'notebook=5.1.*' \
+-    'notebook=5.2.*' \
 -    'jupyterhub=0.7.*' \
 -    'jupyterlab=0.18.*' \
 -    && conda clean -tipsy
 +# Install Jupyter notebook and Hub
 +RUN yes | pip install --upgrade pip
 +RUN yes | pip install --quiet --no-cache-dir \
-+    'notebook==5.1.*' \
++    'notebook==5.2.*' \
 +    'jupyterhub==0.7.*' \
 +    'jupyterlab==0.18.*'
```

Comment:
Should we be switching to `pip` here? Also `pip` is a `conda` package. So it might be better to upgrade `pip` via `conda` if it is needed.

Reply:
Are there ppc64 versions of these packages?

Reply:
Good point! I guess not since this is a recent `notebook` release.

Reply:
Though all the other packages had come from Miniconda for ppc64le before. Maybe we should just drop the `pip` upgrade line. The others (excepting `jupyterhub`) require the `notebook`.

Reply:
We currently don't have a way to build the ppc64 versions ourselves so I'm a bit hesitant to change anything with the setup beside the minor notebook version number in this PR. Issue #472 is open to address ppc64 going forward, and @seth-priya, @vaibhavsood, or anyone can follow-up with other improvements.